### DISCUSSION TITLE
前端开发的瓶颈与未来之路

### DISCUSSION THREAD

**Discussion by giscus** [keelii/fresh-blog#12] [Announcements]
# 前端开发的瓶颈与未来之路

芝兰生于深林，不以无人而不芳，君子修道立德，不为穷困而败节。

https://keelii.com/2020/05/11/frontend-dev-bottleneck-and-future

**Comment by chiehw** [comment]
作者你好啊，我很喜欢你关于 Anders 的那几段，我在文章里引用了一下可以吗？https://juejin.cn/spost/7399273700117479474

**Comment by 2234839** [comment]
关于您文中提到的架构那方面，我认为现在 js 生态有了良好的解决方案，那就是  https://effect.website/
